starter-project/tailwind.config.js

```diff
@@ -0,0 +1,18 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+    './src/pages/**/*.{js,ts,jsx,tsx}',
```

Comment:
We can remove src/pages and components I think since these will all be in app

Reply:
Backlog task added